websites/P/PlanetXO/presence.ts

```diff
@@ -0,0 +1,200 @@
+const presence = new Presence({
+		clientId: "1260572178068406282",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey:
+			"https://cdn.kaneproductions.co.uk/art/PlanetXO/shortlogo.png",
+	};
+
+	if (document.location.hostname === "planetxo.uk") {
+		presenceData.startTimestamp = browsingTimestamp;
+		switch (document.URL.split("/").slice(0, -1).join("/")) {
```

Comment:
Use `document.location.href` instead of `document.URL` for consistency with other presences